library/Network/CloudSeeder/Template.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Network.CloudSeeder.Template 
```

Comment:
Could you add a very small module documentation comment here that notes this module is for parsing the YAML CloudFormation templates?